### ISSUE TITLE
Help not shown

### ISSUE THREAD

**Issue by phuppe** [fgpv-vpgf/fgpv-vpgf#2041]
The Help is no longer showing at 1.7.0.  We are getting a popup with the title, the search box and a close icon.  No body!  It is trying to load a file "http://dev.gcgeo.gc.ca/viz/help/en-CA.md" which is not found.  This file is not in the distribution.  I have found it in the samples download.  I copied the "help" folder to my "viz" folder, but still get the 404 error.  Do I need to create a Mime Type for this extension?  Can you use ".txt" instead of ".md" so that we do not need to ask SSC to add the Mimi Type?

The "help" folder should be in the distribution folder as it is required.

**Comment by milesap** [comment]
Please see: http://fgpv.cloudapp.net/demo/v1.7.0/prod/samples/index-mobile.html

This shows production loading with help shown. I uploaded the v1.7.0 files again from our cloud.

**Comment by milesap** [comment]
Samples are for local development only, they should never be used or copied into a production environment.

**Comment by phuppe** [comment]
Yes, it works.  But it will not work on our servers with ".md" files.  Do I need a Mime Type setup on the servers?  That is why I asked if they could be changed to ".txt" instead which is already supported by the SSC servers.  These ".md" are only text files after all.
I do not consider the Help files samples.  To me, those are integral part of the system.  Unless you think they are incomplete and expect the implementer to complete them.

**Comment by milesap** [comment]
Good points, we're discussing where help files should be placed in the distribution chain and should have this clarified shortly. 

At the moment the file names cannot be changed. They are [hard coded](https://github.com/fgpv-vpgf/fgpv-vpgf/blob/v1.7.0/src/app/ui/help/help.service.js#L204) as `[current language].md`. Our v2.0 release [is similar](https://github.com/fgpv-vpgf/fgpv-vpgf/blob/develop/src/app/ui/help/help.service.js#L211). 

I'm not sure about the `.md` file type not being served on your end. It's technically not an issue to change the file extensions to `.txt`. Perhaps this can be a config property? I'll let the team mull this over. For now, you'll need to serve the `.md` files.

**Comment by phuppe** [comment]
Please let me know as soon as possible about the ".md" vs ".txt' extension as I do not want to ask SSC to add the Mime Type if not necessary.  We currently only have version 1.7.0 on DEV, so we can live with no help until a decision is made.

**Comment by milesap** [comment]
@mweech what do you think about `.md` vs `.txt`?

**Comment by phuppe** [comment]
Just noticed that you are also using the same ".md" files for "About".  So, we would want to apply the decision to that as well.

**Comment by milesap** [comment]
After discussing with @mweech and @AleksueiR:
- Help file extensions will remain as `.md`, please contact SSC to serve these files on your server.
- We are working on having help files bundled separately from the viewer code and made available on our release page - track on #2049.
- There is a configuration property available on v2.0.0 `help.folderName` where you can specify the folder location of your help files
- We will likely hide the help button when this config property is missing (to avoid displaying an empty help box - track on #2048)

If you have any other questions let us know here or on the child issues. Closing for now

**Comment by phuppe** [comment]
OK. I will ask SSC to add the Mime Type.

**Comment by phuppe** [comment]
Now that I have it sowing, I see that you have missing images in several sections
